app/farm/views.py

```diff
@@ -0,0 +1,42 @@
+from django.http import HttpResponse, HttpResponseRedirect
+from django.shortcuts import render, get_object_or_404
+from django.urls import reverse
+
+from .models import Farm
+
+
+def index(request):
+    # return HttpResponse("You're looking at a test farm")
+    farm_list = Farm.objects.order_by('-name')[:5]
```

Comment:
As discussed, just need to chain a `get("user"=request.user)`into this query to just get the farms for the specific farmer.

Reply:
done